frontend/src/components/Toggle/Toggle.tsx

```diff
@@ -58,9 +63,12 @@ export const Toggle = forwardRef<ToggleProps, 'input'>(
       <Flex __css={{ ...styles.overallContainer, ...containerStyles }}>
         {(label || description) && (
           <Box __css={styles.textContainer}>
-            <FormLabel.Label sx={{ ...styles.label, ...labelStyles }}>
-              {label}
-            </FormLabel.Label>
+            <Box display="flex" alignItems="center">
+              <FormLabel.Label sx={{ ...styles.label, ...labelStyles }}>
+                {label}
+              </FormLabel.Label>
+              {labelComponentRight ?? labelComponentRight}
```

Comment:
Is this a typo? What is the intent of this operator? 🤔

Reply:
Seems to be a mistake made by me, I intended to use the [Nullish_coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing) and misread that it would return right_expr if left_expr is not null, otherwise it would return null.

But after reading the docs again I realised it returns left_expr if it's not null and right otherwise. 

I will fix this

Reply:
Fixed to `<labelComponentRight /> ?? null` 
if the labelComponentRight does not exist, it will be null